src/Dialog/Dialog.js

```diff
@@ -10,6 +10,8 @@ import Paper from '../Paper';
 import ReactTransitionGroup from 'react-addons-transition-group';
 
 class TransitionItem extends Component {
```

Comment:
`TransitionItem` and `DialogInline` did not get a test like `Dialog`. I'm commenting in case that wasn't on purpose.